packages/@react-spectrum/calendar/docs/Calendar.mdx

```diff
@@ -0,0 +1,161 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/calendar';
+import i18nDocs from 'docs:@internationalized/date';
+import {HeaderInfo, PropTable, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/calendar/package.json';
+
+```jsx import
+import {Calendar} from '@react-spectrum/calendar';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Date and Time
+keywords: []
+after_version: 3.0.0
+---
+
+# Calendar
+
+<PageDescription>{docs.exports.Calendar.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Calendar']}
+  sourceData={[]} />
+
+## Example
+
+```tsx example
+<Calendar aria-label="Event date" />
+```
+
+## Value
+
+A `Calendar` has no selection by default. An initial, uncontrolled value can be provided to the `Calendar` using the `defaultValue` prop. Alternatively, a controlled value can be provided using the `value` prop.
```

Comment:
It's not really a placeholder though. The current date is always highlighted even if there is a selection. I'm not sure it's necessary since it really has nothing to do with the selection.